src/helper/util.ts

```diff
@@ -254,4 +286,65 @@ export class ZkWasmUtil {
     let signature = await wallet.signMessage(message);
     return signature;
   }
+
+  // For nodejs/server environments only
+  static async loadContextFileFromPath(
+    filePath: string
+  ): Promise<ContextHexString> {
+    if (typeof window === "undefined") {
+      // We are in Node.js
+      const fs = require("fs");
+      //const fs = await import("fs").then((module) => module.promises);
+      return fs.readFile(filePath, "utf8");
+    } else {
+      // Browser environment
+      throw new Error(
+        "File loading in the browser is not supported by this function."
+      );
+    }
+  }
+
+  // For nodejs/server environments only
+  static async loadContexFileAsBytes(filePath: string): Promise<Uint8Array> {
+    const fileContents = await this.loadContextFileFromPath(filePath);
+    return new TextEncoder().encode(fileContents);
+  }
+
+  // Load file for browser environments
+  static async browserLoadFileAsBytes(file: File): Promise<Uint8Array> {
+    if (typeof window === "undefined") {
+      // We are in Node.js
+      throw new Error(
+        "File loading in Node.js is not supported by this function."
+      );
+    }
+    return new Promise((resolve, reject) => {
+      const reader = new FileReader();
+
+      reader.onload = function () {
+        // Validate that the file size is a multiple of 8 bytes (64 bits)
+        if (
+          reader.result &&
+          ZkWasmUtil.validateContextBytes(
+            new Uint8Array(reader.result as ArrayBuffer)
+          )
+        ) {
+          resolve(new Uint8Array(reader.result as ArrayBuffer));
+        } else {
+          reject("File size must be a multiple of 8 bytes (64 bits)");
+        }
+      };
+
+      reader.onerror = function (error) {
+        reject(error);
+      };
+
+      reader.readAsArrayBuffer(file);
+    });
+  }
+
+  // Validate bytes are a multiple of 8 bytes (64 bits)
+  static validateContextBytes(data: Uint8Array): boolean {
```

Comment:
Better also add 4k size limit check in F/E？
And better add validating into nodejs function loadContexFileAsBytes?